openshift/scripts/ocp-template-install.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+function usage {
+    echo "Usage: ./oc-template-install.sh <api_key> <resource_group_id> <cluster_name> <template-file>"
```

Comment:
I think we don't need to include the template as an arg. Let's point to the template in `/openshift/templates/<whateverthenamewillbe.yml>`

Reply:
Are we assuming execution from the root directory, i.e. `./openshift/scripts/ibm-svs-template-install.sh`?